src/org/linphone/InCallActivity.java

```diff
@@ -594,9 +594,10 @@ public void callEncryptionChanged(LinphoneCore lc, final LinphoneCall call, bool
 			videoCallFragment = (VideoCallFragment) callFragment;
 
 			if(call != null && call.getDirection() == CallDirection.Outgoing && isConnecting(call.getState())){
-				if(call.getState() == State.OutgoingRinging)
-				{
-					labelRingingView.setText("Ringing...");
+				if(call.getState() == State.OutgoingRinging) {
+					String ringingText = "Ringing... \n" + call.getRemoteAddress().asStringUriOnly();
+					labelRingingView.setLines(4);
+					labelRingingView.setText(ringingText);
 				}
```

Comment:
Add
else{
labelRingingView.setText(labelRingingView.getText() + "\n" +call.getRemoteAddress().asStringUriOnly() );
}
so it will show address while connecting as well (before getting ringing from remote end)